chia/cmds/init_funcs.py

```diff
@@ -102,14 +106,20 @@ def check_keys(new_root: Path, keychain: Optional[Keychain] = None) -> None:
     if "pool" not in config:
         config["pool"] = {}
     if "xch_target_address" not in config["pool"]:
-        print(f"Setting the xch destination address for coinbase reward to {all_targets[0]}")
+        print(f"Setting the xch destination address for pool reward (7/8 for solo only) to {all_targets[0]}")
         config["pool"]["xch_target_address"] = all_targets[0]
+        updated_target = True
     elif config["pool"]["xch_target_address"] not in all_targets:
         print(
             f"WARNING: using a pool address which we don't have the private"
             f" keys for. We searched the first {number_of_ph_to_search} addresses. Consider overriding "
             f"{config['pool']['xch_target_address']} with {all_targets[0]}"
         )
+    if updated_target:
+        print(
+            f"To change the XCH destination addresses, edit the `xch_target_address` entries in"
+            f" {(new_root / 'config.yaml').absolute()}."
```

Comment:
I think this is missing the 'config' directory, but could be mistaken.

```suggestion
            f" {(new_root / 'config/config.yaml').absolute()}."
```